apps/flash-liquidator/src/hooks/useSendFlashLiquidateTransaction.ts

```diff
@@ -53,12 +53,15 @@ export const useSendFlashLiquidateTransaction = (
 
   const args = useBestLiquidationArgs(liquidationPair, { receiver: userAddress })
 
+  const gasLimit = 550_000n + BigInt(((liquidationPair.swapPath.length - 1) / 2) * 150_000)
```

Comment:
It's probably best to have this be conditional; get a gas estimate, get a hardcoded fallback like is being done now, and if the gas estimate is lower than the fallback, then use the fallback (functionally a minimum).